lib/storage/src/content_manager/consensus_manager.rs

```diff
@@ -609,20 +608,20 @@ impl<C: CollectionContainer> ConsensusManager<C> {
             // Okay if on the same term and have at least the specified commit
             let is_ok = state.term == term && state.commit >= commit;
             if is_ok {
-                return true;
+                return Ok(());
             }
 
-            // Fail if on a different term
-            let is_fail = state.term != term;
+            // Fail if on a newer term
+            let is_fail = state.term > term;
```

Comment:
I changed the condition:

```diff
-let is_fail = state.term != term;
+let is_fail = state.term > term;
```

This is on a remote node, on which we wait for it to reach some consensus (commit + term).

Before, waiting on a consensus commit always failed right away if the current term is not exactly the same as got requested.

After this change, it won't fail right away anymore if the current term has not caught up yet.  This gives the node some time to reach the same term if a reelection happened.

@ffuugoo Maybe you have some good insight on this specifically.